tests/t0118_gateway_car_test.go

```diff
@@ -44,10 +44,87 @@ func TestGatewayCar(t *testing.T) {
 						Equals("none"),
 				).Body(
 					IsCar().
-					 	HasRoot(fixture.MustGetCid("subdir", "ascii.txt")).
-						HasBlock(fixture.MustGetCid("subdir", "ascii.txt")).
+					 	HasRoot(fixture.MustGetCid()).
+						HasBlocks(
+							fixture.MustGetCid(),
+							fixture.MustGetCid("subdir"),
+							fixture.MustGetCid("subdir", "ascii.txt"),
+						).
 						Exactly().
-						InThatOrder(),	
+						InThatOrder(),
+				),
+		},
+		{
+			Name: "GET with ?format=car&car-scope=block params returns expected blocks",
+			Hint: `
+				car-scope=block should return a CAR file with only the root block and a
+				block for each optional path component.
+			`,
+			Request: Request().
+				Path("ipfs/%s/subdir/ascii.txt", fixture.MustGetCid()).
```

Comment:
```suggestion
				Path("ipfs/{{cid}}/subdir/ascii.txt", fixture.MustGetCid()).
```

Note we introduced templates to work around the collision between `%verbs` and HTML escape codes, etc.
https://github.com/ipfs/gateway-conformance/pull/41

You'll have to rename these formatting.